addons/website_mass_mailing/static/src/js/website_mass_mailing.dialog.js

```diff
@@ -0,0 +1,34 @@
+odoo.define('website_mass_mailing.dialog', function (require) {
+
+var core = require('web.core');
+var Dialog = require('web.Dialog');
+
+var _t = core._t;
+
+return Dialog.extend({
+    /**
+     * @override
+     */
+    willStart: function () {
+        var self = this;
+        self.$content.attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
+        return this._super(this, arguments).then(function () {
+            self.$modal.addClass('o_newsletter_modal');
+            self.$modal.find('.modal-dialog').addClass('modal-dialog-centered');
+        });
+    },
+    /**
+     * @override
+     */
+    destroy: function () {
+        var $modal = this.$modal;
+        if ($modal) {
+            $modal.find('.o_fake_editable')
+                .removeClass('o_fake_editable')
+                .removeAttr('contenteditable');
```

Comment:
Why is this necessary ?

Reply:
`wysiwyg_multizone` will add class `o_fake_editable` and `contenteditable` attribute to all direct children elements of the `oe_structure` on `_onActivateSnippet`. 
see: [wysiwyg_multizone](https://github.com/odoo/odoo/blob/master/addons/website/static/src/js/editor/wysiwyg_multizone.js#L495)
so before saving content to the database, we have to remove `.o_fake_editable` and `contenteditable` attribute. otherwise, even in read-only mode, the popup will be editable.

Reply:
Yes, but my question was "why is not the editor already removing all of this stuff before saving by itself?"

Reply:
Because in our case `oe_structure` element is in the Dialog and on dialog 'closed' Dialog will be removed from the DOM. so on dialog 'closed' currently, the DOM of the popup (it content `contenteditable` and `.o_fake_editable` at that time) is saved as `.data()` on the related snippets as per our previous  [conversation](https://github.com/odoo/odoo/pull/29353#discussion_r284670478) and on `_saveElement` we will set content for a particular newsletter.

Reply:
Can't you call an editor function / trigger an event which will force the editor to remove those classes by himself on dialog close ? I am worried about duplicated code here... especially since the editor is being refactored :/

Reply:
I've checked but there is no direct function/trigger available for this kind of thing. so I've done this by adding the custom event in `EditorMenu` see [commit](https://github.com/odoo-dev/odoo/commit/cb8cf6a5b85e5d9f42ee977d97e0d4649e11c791)

could you please give your opinion on this?

Reply:
Seems fine. We are merging this for the last saas of the year anyway and everything will be broken by the new editor so we will have to adapt afterwards... :|

Reply:
Ok, I squash the last commit. And the branch is green after a long time :yum: